signposting.md

```diff
@@ -0,0 +1,123 @@
+# Signposting Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## The /signposting is used to implement Signposting.
+
+Signposting is an approach to make the scholarly web more friendly to machines. It uses Typed Links as a means to clarify patterns that occur repeatedly in scholarly portals. For resources of any media type, these typed links are provided in HTTP Link headers. For HTML resources, they may additionally be provided in HTML link elements. More information at [Signposting the Scholarly Web](https://signposting.org/) 
+
+DSpace supports the FAIR signposting profile at level 2, see https://signposting.org/FAIR/
+
+## Main Endpoint
+```/signposting/linksets```
+
+As we don't have yet an use case to iterate over all the existent linksets the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+## Single Item linkset 
+
+### in application/linkset+json serialization
+```/signposting/linksets/<:uuid>/json```
+
+This endpoint provides typed links of item having uuid specified in input, in application/linkset+json serialization
+
+```json
+{
+  "linkset": [
+    {
+      "item": [
+        {
+          "href": "https://{dspace.url}/bitstream/123456789/7/1/test.txt",
+          "type": "text/plain"
+        },
+        {
+          "href": "https://{dspace.url}/bitstream/123456789/7/1/test.pdf",
+          "type": "application/pdf"
+        }
+      ],
+      "author": [
+        {
+          "href": "http://orcid.org/0000-0002-3748-8359"
+        },
+        {
+          "href": "https://isni.org/isni/0000002251201436"
+        }
+      ],
+      "cite-as": [
+        {
+          "href": "https://doi.org/10.1007/978-3-642-35233-1_18"
+        }
+      ],
+      "anchor": "https://{dspace.url}/handle/123456789/7"
+    },
+    {
+      "anchor": "https://{dspace.url}/bitstream/123456789/7/1/test.txt",
```

Comment:
These extra "anchor" sections for each bitstream are not appearing for me when I test the backend PR.  Are they supposed to appear?

Reply:
No, they aren't supposed to appear. Documentation has been updated